src/parser/TerraformListener.js

```diff
@@ -31,10 +35,35 @@ class TerraformListener extends antlr4.tree.ParseTreeListener {
     this.fieldsTree = [];
   }
 
+  addVariable() {
+    this.currentVariable.path = this.currentFile.path;
+    this.variables.push(this.currentVariable);
+    this.currentVariable = null;
+    this.currentVariableField = null;
+  }
+
   getAttributeDefinition(object, name) {
     return object.definition?.definedAttributes.find((def) => def.name === name) || null;
   }
 
+  isVariable() {
+    return (this.currentBlockType === 'variable' || this.currentBlockType === 'output');
+  }
+
+  // Convert the type to match one of a variable
```

Comment:
remove comment or transform to documenation.

Reply:
Done